pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1124,26 +1145,7 @@ func nodesWherePreemptionMightHelp(nodes []*v1.Node, failedPredicatesMap FailedP
 		// significant overhead.
 		// Also, we currently assume all failures returned by extender as resolvable.
 		for _, failedPredicate := range failedPredicates {
```

Comment:
for cleanup, prefer to have a helper func to include this `for` loop.

Reply:
Done